packages/store/src/internal/validation/config-validator.ts

```diff
@@ -0,0 +1,32 @@
+import { Inject, Injectable } from '@angular/core';
+
+import {
+  CONFIG_MESSAGES as MESSAGES,
+  VALIDATION_CODE as CODE
+} from '../validation/validation-messages.config';
+import { NG_DEV_MODE, NGXS_TEST_MODE, NgxsConfig } from '../../symbols';
+
+@Injectable()
+export class ConfigValidator {
+  constructor(
+    @Inject(NGXS_TEST_MODE) public isTestMode: Function,
+    @Inject(NG_DEV_MODE) public isNgDevMode: Function,
+    private _config: NgxsConfig
+  ) {}
+
+  private get isIncorrectProduction(): boolean {
+    return !this.isTestMode() && !this.isNgDevMode() && this._config.developmentMode;
```

Comment:
I didn't see this change before either.
Could you move the `!this.isTestMode()` back to being a guard clause as the first line in `verifyDevMode`?

Reply:
ok

Reply:
done